### DISCUSSION TITLE
Autofill dell'anagrafica nella proforma creata

### DISCUSSION THREAD

**Discussion by simsac81** [fattureincloud/api#181] [Q&A] [answered]
Salve,
nella precedente versione delle API usando in creazione proforma il campo autocompila_anagrafica valorizzato con "true" nella proforma veniva compilata automaticamente l'anagrafica del cliente con i dati presenti su Fatture in Cloud.

Nella nuova versione delle API come posso ottenere lo stesso risultato?

Grazie

**Comment by simsac81** [comment, marked as answer]
Ho trovato sulla documentazione il "warning" riguardo questo aspetto e vedo "We're currently working to add something similar on the APIv2, so stay tuned!".

Quando pensate di rilasciare questa funzionalità? Sarebbe importante per evitare di dover fare 2 richieste per creare una proforma.

Grazie

**Reply by valmoz** [reply]
Buongiorno,
al momento non abbiamo una data di rilascio per questa funzionalità. Aggiorneremo la pagina quando avremo più informazioni in merito.

Per il momento, le possibilità sono le seguenti:

- [Creare il cliente](https://developers.fattureincloud.it/docs/guides/client-creation/) in anagrafica (se non già esistente) con una Create Client, e poi passare l'ID del cliente nella richiesta Create Issued Document
- Evitare del tutto l'inserimento del cliente in anagrafica, inserendo direttamente i dettagli del cliente e omettendo il campo "id"

Può trovare maggiori dettagli qui: [Invoice Creation - The Customer](https://developers.fattureincloud.it/docs/guides/client-creation/)

Grazie
Mauro
